testing/nlohmann-json/APKBUILD

```diff
@@ -1,7 +1,7 @@
 # Contributor: Nick Black <dankamongmen@gmail.com>
 # This is a headers-only implementaiton; there are no libraries.
 pkgname=nlohmann-json-dev
-pkgver=3.5.0
+pkgver=3.6.1
 pkgrel=1
```

Comment:
When updating the version `pkgrel` should be reset to 0

Reply:
Thanks! Fixed.